lighthouse-core/test/audits/seo/meta-description-test.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright 2017 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('../../../audits/seo/meta-description.js');
+const assert = require('assert');
+
+/* eslint-env mocha */
+
+describe('SEO: description audit', () => {
+  it('fails when HTML does not contain a description meta tag', () => {
+    return assert.equal(Audit.audit({
```

Comment:
it would be good to assert that the audit is also returning a `debugString` (or a `displayValue`, depending on how the comment above shakes out :)

Reply:
will do 👍